.circleci/config.yml

```diff
@@ -442,45 +442,137 @@ jobs:
+      - store_test_results:
+          path: backstop/report
+      - store_artifacts:
+          path: backstop
+
+  backstop_reference:
+    parameters:
+      target:
+        type: string
+        default: prod
+      list:
+        type: string
+        default: all
+      viewport:
+        type: string
+        default: all
+    resource_class: xlarge
+    docker:
+      - image: backstopjs/backstopjs
+    steps:
+      - checkout
+      - run:
+          name: Capture Backstop Reference Screenshots
+          working_directory: backstop
+          command: |
+            backstop --config=backstop.js reference --target=<< parameters.target >> --list=<< parameters.list >> --viewport=<< parameters.viewport >>
+      - run:
+          name: Archive Reference Screenshots
+          command: |
+            apt-get update
+            apt-get install zip -y
+            zip -r references.zip backstop/reference
+      - store_artifacts:
+          path: references.zip
+          destination: backstop-<< parameters.target >>-latest
+      - store_artifacts:
+          path: backstop/reference
+      - run:
+          name: Save Build Number
+          command: |
+            curl --request POST \
+              --fail \
+              --url "https://circleci.com/api/v2/project/gh/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/envvar" \
+              --header "Circle-Token: $CIRCLE_PERSONAL_TOKEN" \
+              --header 'content-type: application/json' \
+              --data "{\"name\":\"BACKSTOP_REF_<< parameters.target >>\",\"value\":\"$CIRCLE_BUILD_NUM\"}"
+
+  backstop_test:
+    parameters:
+      reference:
+        type: string
+        default: prod
+      target:
+        type: string
+        default: test
+      list:
+        type: string
+        default: all
+      viewport:
+        type: string
+        default: all
+      tugboat:
+        type: string
+        default: ""
+    resource_class: xlarge
+    docker:
+      - image: backstopjs/backstopjs
+    steps:
+      - checkout
+      - run:
+          name: Verify Artifacts
+          command: |
+            apt-get update
+            apt-get install jq -y
+            BACKSTOP_REF="BACKSTOP_REF_<< parameters.reference >>"
+            if [ "${!BACKSTOP_REF}" == "" ]; then
+              echo "No artifacts found for << parameters.reference >>"
+              exit 1
+            fi
+            STOPPED_AT=$(curl --request GET \
+              --fail \
+              --url https://circleci.com/api/v2/project/gh/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/job/${!BACKSTOP_REF} \
+              --header "Circle-Token: $CIRCLE_PERSONAL_TOKEN" | jq -r '.stopped_at')
+            LAST_SUCCESS=$(date -d "$STOPPED_AT" +%s)
+            STALE_TIME=$(date -d "-25 hours" +%s)
+            if [ $LAST_SUCCESS -lte $STALE_TIME ]; then
+              echo "Reference images are stale, please re-run the backstop_reference job"
+              exit 1
```

Comment:
I think this should be just a warning and should not fail the job. Its OK if we use references that are a few days out of date.